docs/fundamentals/layouts/grid.md

```diff
@@ -1,52 +1,9 @@
-# Grid
+# Flexbox grid
 
-**Layout principles** allow the arrangement of elements in an interface.
-They help developers and designers to ensure consistency, efficiency,
-and effective communication in the user experience.
+The **flexbox grid** is used to create layouts of any complexity using a 12-column system, six responsive [breakpoints](./breakpoints.md), and a wide range of predefined utility classes. The grid system is based on the well-known and beloved Bootstrap grid.
 
 ## Usage ---
```

Comment:
@panch1739 Since we are now left with a developer only docs here in this chapter, we could get rid of the `usage` and `code` tabs and instead have a better right side navigation tree.